apps/portals/playwright.config.ts

```diff
@@ -0,0 +1,93 @@
+import { defineConfig, devices } from '@playwright/test'
+import dotenv from 'dotenv'
+import path from 'path'
+
+const baseURL = 'http://localhost:3000'
+export const viewports = {
+  expanded: { width: 1600, height: 1200 },
+  collapsed: { width: 1280, height: 720 },
+}
+export const defaultExpectTimeout = 10 * 1000
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+dotenv.config()
+dotenv.config({
+  path: path.resolve(`src/configurations/${process.env.PORTAL!}/.env`),
+})
```

Comment:
Read environment variables from the the Portal's configuration directory `.env` file, e.g. VITE_PORTAL_NAME